src/Components/MessageState/MessageState.js

```diff
@@ -1,20 +1,29 @@
-import { EmptyState, EmptyStateBody, EmptyStateIcon, EmptyStateVariant } from '@patternfly/react-core/dist/esm/components/EmptyState/index';
+import {
+    EmptyState,
+    EmptyStateBody,
+    EmptyStateIcon,
+    EmptyStateVariant,
+    EmptyStateHeader,
+    EmptyStateFooter
+} from '@patternfly/react-core/dist/esm/components/EmptyState/index';
```

Comment:
Let's make sure this migration does not change imports from relative path to absolute path. Ref: https://redhat-internal.slack.com/archives/C02T90PLJ5V/p1705403328061789

Reply:
The migration did not change any relative imports to absolute, all absolute imports were absolute before or are completely new (like SkeletonTable), I would suggest changing all absolute imports for the whole app in a separate PR, since this one is already quite sizable.

Reply:
Yeah, I noticed now that malware uses an absolute path for most of the PF imports. But still, better to disable that eslint rule within this PR and avoid the new usages of absolute path wherever possible. We can work on the remaining already existing imports later.